src/Dfe.EarlyYearsQualification.Web/Views/Home/Index.cshtml

```diff
@@ -9,7 +9,7 @@
         <h1 class="govuk-heading-xl">@Model.Header</h1>
         <div id="pre-cta-content" class="govuk-body">@Html.Raw(Model.PreCtaButtonContent)</div>
         @* SonarLint says turn this into a button, or give it onKeyPress|onKeyDown|onKeyUp attribute -- but this construct is mandated by Gov UK guidelines *@
-        <a href="@Url.Action("StartNew", "Questions")" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button" id="start-now-button">
+        <a data-testid="start-button" href="@Url.Action("StartNew", "Questions")" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button" id="start-now-button">
```

Comment:
Just a question. This already has an id, could you not use the locator function in the smoke test and pass it this id instead?